Solaris/MoonPhase.php

```diff
@@ -136,6 +139,7 @@ public function __construct($date = null)
 		$this->age = $synmonth * $this->phase;							// Age of moon (days)
 		$this->distance = $MoonDist;									// Distance (kilometres)
 		$this->diameter = $MoonAng;										// Angular diameter (degrees)
+		$this->moondegree = round($MoonAge, 0);								//Age of the Moon in degrees
```

Comment:
I don't think this should be rounded, as we lose accuracy. Projects can round the result themselves if they need to.